### PR TITLE
win32: support `--alpha=yes`

### PR DESCRIPTION
**Pull request by na-na-hi**
This adds support for `--alpha=yes` for `--gpu-api=d3d11` (with `--d3d11-flip=no`) and `--gpu-api=opengl` on win32.

### CODE REVIEW

File: video/out/opengl/context_win.c [hunk 1]
```diff
@@ -96,6 +96,7 @@ static bool create_dc(struct ra_ctx *ctx)
 
     pfd.iPixelType = PFD_TYPE_RGBA;
     pfd.cColorBits = 24;
+    pfd.cAlphaBits = 8;
```
Comment (sfan5) [inline video/out/opengl/context_win.c:99]: couldn't this cause less optional formats to be picked?
I would put this behind a check

**Comment by Dudemanguy** [comment]
Would it be possible to make this work on runtime? Wayland uses the `update_render_opts` function pointer so the compositor's opaque region setting can be flipped on and off although it seems gpu-next has a bug since after you set alpha it never seems to unset itself. It works in vo_gpu though. X11 on mesa is still bugged but allegedly it works with NVIDIA.

**Comment by Dudemanguy** [comment]
Huh really? I have not actually looked at the libplacebo code here but it works in wayland so I just assumed libplacebo was already handling it. If not, then no need to worry about it.

**Comment by Dudemanguy** [comment]
Someone mentioned this to me in IRC and I have not actually verified it myself, but it looks like libplacebo does set `compositeAlpha` here: https://github.com/haasn/libplacebo/blob/3ba18d5d84738c663e16d7699342102e30e2fbc5/src/vulkan/swapchain.c#L468

I assume that works on wayland but not windows for some reason (perhaps for the reason you just stated)? I think it makes since for an API user to be able to force this in any case. Doesn't seem like you currently can.

**Review by Dudemanguy — APPROVED** [review]
In any case, setting the compositor transparency on our side is probably a good thing even if it doesn't work because of some driver/who-knows-what setting.

**Pull request merged**